homeassistant/components/weather/__init__.py

```diff
@@ -83,6 +86,11 @@ def attribution(self):
         return None
 
     @property
+    def forecast(self):
```

Comment:
New properties added to the base class should also be added to the demo weather platform and should be covered by tests.